packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -301,7 +302,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           onDismiss={ this._onDismiss }
           onPositioned={ this._onPositioned }
         >
-          <div style={ { width: this._dropDown.clientWidth - 2 } }>
+          <div style={ { width: dropdownWidth || this._dropDown.clientWidth - 2 } }>
```

Comment:
Really we need to give control in a lot more places to the callout for a variety of things. It's got like 3 different boxes that all need to be in sync. Someone wants to make rounded edges and it's a bear.